public/app/plugins/panel/geomap/globalStyles.ts

```diff
@@ -78,5 +78,8 @@ export function getGlobalStyles(theme: GrafanaTheme2) {
     .ol-attribution:not(.ol-collapsed) {
       background-color: ${theme.colors.background.secondary}; // rgba(255,255,255,0.8);
     }
+    .panel-content {
```

Comment:
🤔 I think this gets applied to ALL panels, not just the geomap one.

Reply:
Yes -- it is equivalent to dynamically changing:
https://github.com/grafana/grafana/blob/dashboard-list-panel/public/sass/pages/_dashboard.scss#L45

after the geomap panel loads

Reply:
ugh, I thought it's applied where it's being used.. 👀

Reply:
This may or may not be the worst thing given in original issue user also mentions other cases where overflow being disabled causing broken functionality 

See https://github.com/grafana/grafana/issues/54094 
```
Anything else we need to know?:
Issue is probably similar for other panels. 
I have created dropdown navigation in text panel and it also stopped working as expected. 
It was able to overflow beyond text panel but from version 9.03 up dropdown is stopped at panel border.
```

Reply:
Yes, I saw it's being referenced in other issues too, but not sure why it was needed in the first place.. ie - what issues the revert is going to cause